Makefile

```diff
@@ -158,6 +158,12 @@ cover: nocover
 	mv cover-nogen.out cover.out
 	go tool cover -html=cover.out -o cover.html
 
+.PHONY: cover-otel
+cover-otel:
+	@echo pre-compiling OTEL tests
+	@cd ${OTEL_COLLECTOR_DIR} && time go test -i $(shell cd ${OTEL_COLLECTOR_DIR} && go list ./...)
```

Comment:
I don't think pre-compiling helps much or even needed when running a single `go test`. It made more sense when using cover.sh because that one was testing each module independently.